actors/builtin/miner/miner_actor.go

```diff
@@ -485,9 +485,16 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 
 	store := adt.AsStore(rt)
 	var st State
+	var err error
 	newlyVested := big.Zero()
+	feeDebtPenalty := abi.NewTokenAmount(0)
 	rt.State().Transaction(&st, func() {
-		verifyPledgeMeetsInitialRequirements(rt, &st)
+		newlyVested, err = st.UnlockVestedFunds(store, rt.CurrEpoch())
+		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to vest funds")
+		unlockedBalance := st.GetUnlockedBalance(rt.CurrentBalance())
+
+		feeDebtPenalty = verifyPledgeRequirementsAndRepayDebts(rt, &st)
```

Comment:
It took me quite some reading to understand `feeDebtPenalty`. A name like `feeToBurn` would help.